riptide-stream/src/test/java/org/zalando/riptide/stream/EnforceCoverageTest.java

```diff
@@ -37,14 +37,14 @@
 
     @Test
     public void shouldUseStreamsConstructor() {
-        new Streams();
+        assertNotNull(new Streams());
     }
 
     @Test
     public void shouldUseStreamConverterConstructor() {
-        Streams.streamConverter();
-        Streams.streamConverter(null);
-        Streams.streamConverter(null, null);
+        streamConverter();
```

Comment:
Because I have the weird opinion that a unit test should use the component or testing instead of some hiding support class.